akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -6,6 +6,7 @@ package akka.cluster.sharding
 import java.net.URLEncoder
 import java.util.Optional
 import java.util.concurrent.ConcurrentHashMap
+import java.util.function.Function
```

Comment:
This is a bit scary since we are used to that `Function` is the Scala function.
we often rename that:
```
import java.util.function.{ Function => JFunction}
```